.github/ISSUE_TEMPLATE/add-content.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Community contribution - add new content
+about: Use this issue to submit new content to the Vocabularies
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Instructions
+
+
+**0. Create the initial issue.**
+
+Please create the initial issue that briefly describes the content of intended contribution and rationale behind it.
+
+**1. Read the documentation.**
+
+Please familiarize yourself with the documentation [here](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Community-contribution) prior to initiating the submission.
+
+**2. Choose the category of submission.**
+
+Choose the category of your submission and download the corresponding template:
+
+- Template 1: Add non-standard concept(s) to an existing vocabulay (with or without synonyms or mappings);
+- Template 2: Add synonym(s) to existing concept(s);
+- Template 3: Add mapping to existing concept(s);
+- Template 4: Add new non-standard vocabulary (with or without mappings).
+
+**3. Fill the template.**
+
+Please dowload the corresponding template from [this folder](https://drive.google.com/drive/u/1/folders/1D7hSGKQkWgJCAKXDNOjfoFcnRca0-57w) and fill it. Instructions and examples are provided inside. If questions occurr use comments in the current issue to contact the Vocabulary Team. Make sure you filled the checklist!
+
+**4. Upload the template.**
+
+Please go to [this folder](https://drive.google.com/drive/u/1/folders/1wXB3CK08-sPa0EdyLlIL8jHaBkuH7k3g), create a subfolder with a short name of your contribution and upload your contribution there.
```

Comment:
@aostropolets What if they name a folder using the github issue id?

Reply:
renamed to issue#+contribution name+author name